content/warnings/dont-call-proptypes.md

```diff
@@ -1,73 +1,72 @@
 ---
-title: Don't Call PropTypes Warning
+title: Don't Call PropTypes Uyarısı
 layout: single
 permalink: warnings/dont-call-proptypes.html
 ---
 
-> Note:
+> Not:
 >
-> `React.PropTypes` has moved into a different package since React v15.5. Please use [the `prop-types` library instead](https://www.npmjs.com/package/prop-types).
+> `React.PropTypes` React v15.5 sürümünden itibaren farklı bir pakete taşındı. Lütfen onun yerine [`prop-types` kütüphanesini](https://www.npmjs.com/package/prop-types) kullanın.
 >
->We provide [a codemod script](/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes) to automate the conversion.
+>Dönüşümü otomatikleştirmek için [codemod scripti](/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes) sunuyoruz.
 
-In a future major release of React, the code that implements PropType validation functions will be stripped in production. Once this happens, any code that calls these functions manually (that isn't stripped in production) will throw an error.
+React'in gelecekteki büyük sürümünde, PropType doğrulaması yapan kod bloğu ayrıştırılacak. Bu olduğu zaman, bu kodun elle çağırıldığı yerler hata verecek.
 
-### Declaring PropTypes is still fine {#declaring-proptypes-is-still-fine}
+### PropTypes tanımlamak hala iyi {#declaring-proptypes-is-still-fine}
 
-The normal usage of PropTypes is still supported:
+PropTypes'ın normal kullanımı hala destekleniyor:
 
 ```javascript
 Button.propTypes = {
   highlighted: PropTypes.bool
 };
 ```
 
-Nothing changes here.
+Burada bir şey değişmiyor.
 
-### Don’t call PropTypes directly {#dont-call-proptypes-directly}
+### PropTypes'ı direkt olarak çağırmayın {#dont-call-proptypes-directly}
 
-Using PropTypes in any other way than annotating React components with them is no longer supported:
+PropTypes'ı React komponentlerini annotate ederek kullanmanın dışındaki yollar desteklenmiyor:
```

Comment:
> komponent -> bileşen

Also, we can translate `annotate` as `aciklamak`. So, this might be better:

> PropTypes'i, React bileşenlerini açıklamak dışında başka bir şekilde kullanmak artık desteklenmemektedir.

Reply:
Fixed